src/pages/Messages/_Star.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Tooltip from '../../components/Tooltip'
+import { Button } from 'antd'
+import {deleteStar, unstarMessage, starMessage, submitStar} from "../../actions/starMessage"
```

Comment:
Action imports should come after library imports, so in this case this should move up to go after `react-redux`. Also add an empty space on the inside of the `{}` marks. Finally, list the destructured parameters in alphabetical order:

```
import { deleteStar, starMessage, submitStar, unstarMessage } from "../../actions/starMessage"
```